components/cards/ActionForm.tsx

```diff
@@ -276,8 +276,12 @@ export function ActionForm(props: ActionFormProps) {
         parcelId: assetId,
         ownerId,
       });
+      const root = await geoWebContent.raw.getPath("/", {
+        parcelId: assetId,
+        ownerId,
+      });
 
-      setRootCid(newRootCid.toString());
+      setRootCid(root?.basicProfile ? newRootCid.toString() : null);
```

Comment:
What if `mediaGallery` exists but `basicProfile` doesn't? We should assume other Cadastres can make writes that we don't know about.

Reply:
It's unclear to me from looking at the code below what a missing `basicProfile` but an existing `mediaGallery` would do.

Reply:
If `basicProfile` is not defined here it means `resolveRoot` returned a CID pointing to an empty object because the edit failed.
By setting `rootCid` to null we show a "Not Available" message and the user can retry to edit later on while preserving the `/mediaGallery` path, otherwise if we check if the media gallery exists here the root CID would just point to an empty object.

Reply:
The root CID would not be an empty object if `/mediaGallery` exists, but `rootCid` would still be set to null here in that case.

Reply:
Oh right, but I think `basicProfile` would be defined too because it was just edited and committed.
We can check for it anyway for clarity and to be sure.

Reply:
Right, that is if the content was created with this Cadastre. But this is more about being interoperable with other cadastres. The schema allows for only one of them to exist.

Reply:
Yes now I get it what you mean before, thanks.